### ISSUE TITLE
NullReferenceException on upgrade

### ISSUE THREAD

**Issue by ayan4m1** [KSP-CKAN/CKAN#1922]
May be related to #1381 but different progression. Possibly related to this specific mod update, but we should be able to handle the error better.

**CKAN Version:** 
1.20.1-0-gd41fe54

**Operating System:** 
Win10 x64

**The issue you are experiencing:** 

```
************** Exception Text **************
System.NullReferenceException: Object reference not set to an instance of an object.
   at CKAN.Main._MarkModForUpdate(String identifier)
   at CKAN.Util.Invoke[T](T obj, Action action)
   at CKAN.Main.PostInstallMods(Object sender, RunWorkerCompletedEventArgs e)
   at System.ComponentModel.BackgroundWorker.OnRunWorkerCompleted(RunWorkerCompletedEventArgs e)
```

**How to recreate this issue:** 
- Install TextureReplacer 2.5.0.
- Add upgrade for TextureReplacer 2.5.1
- On install, exception is thrown in GUI

**Comment by politas** [comment]
Can you try uninstalling Texture Replacer before installing the new version?

**Comment by ayan4m1** [comment]
Yep, fresh install of 2.5.1 with no 2.5.0 doesn't throw the exception.

**Comment by politas** [comment]
Cool. If we could make upgrades like this and those in #1864 "just work" it would make things much easier for everyone.

**Comment by HebaruSan** [comment]
Should be fixed by #2184, as PostInstallMods no longer does this. If there's still a problem, it'll look different and need fresh investigation. Closing.
